rules/apt/apt_gallium.yml

```diff
@@ -0,0 +1,58 @@
+action: global
+title: GALLIUM artefacts
+id: 440a56bf-7873-4439-940a-1c8a671073c2
+status: experimental
+description: Detects artefacts associated with activity group GALLIUM - Microsoft Threat Intelligence Center indicators released in December 2019.
+author: Tim Burrell
+references:
+    - https://www.microsoft.com/security/blog/2019/12/12/gallium-targeting-global-telecom/
+    - https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dn800669(v=ws.11)
+tags:
+    - attack.credential_access
+    - attack.command_and_control
+falsepositives:
+    - unknown
+level: high
+---
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    exec_selection:
+        Hashes:
```

Comment:
@thomaspatzke : do we use the `Hashes` field from Sysmon or do we always require the values to be split up into their respective hash type?

Reply:
My reference for using Hashes was: https://github.com/Neo23x0/sigma/wiki/Taxonomy

Reply:
We use Hashes, as this is how the hashes appear in their events.